libs/core/src/lib/wizard/wizard.component.ts

```diff
@@ -198,14 +204,21 @@ export class WizardComponent implements AfterViewInit, OnDestroy {
 
     /** @hidden */
     private _getDialogOffset(): number {
-        const dialogBody = this._elRef.nativeElement.parentElement;
         let retVal = 0;
-        if (dialogBody.tagName.toLowerCase() === 'fd-dialog-body') {
-            if (dialogBody.querySelector('.' + 'fd-title--h2')) {
-                retVal = dialogBody.querySelector('.' + 'fd-title--h2').offsetHeight;
+        if (this._dialogBodyComponent) {
+            const dialogBody = this._dialogBodyComponent.elementRef().nativeElement;
+            if (dialogBody.tagName.toLowerCase() === 'fd-dialog-body') {
+                const dialogBodyStyle = dialogBody.style;
+                dialogBodyStyle.paddingTop = '0';
+                dialogBodyStyle.paddingBottom = '0';
+                dialogBodyStyle.overflowY = 'hidden';
+                if (dialogBody.querySelector('.' + 'fd-title--h2')) {
```

Comment:
Get element once and set it to some const. Also, no need to contact the string here